docs/wallet.md

```diff
@@ -224,3 +224,17 @@ Creating a new password will overwrite any old one stored on your machine. Make
 3. On the next screen select “Create new password”
 4. Follow the instructions and create a new password
 5. Sign in using your new password
+
+### CLI tool for wallet encryted file (password) recovery:
+
+The mnemonics that are stored in the local password protected file can also be decrypted and recovered through a simple CLI tool, nym-wallet-recovery-cli. 
```

Comment:
A few points:
* add surrounding `` to 'nym-wallet-recovery-cli' - this is the standard for when we're explicitly referring to binary names in the rest of the docs, so lets keep that style consistent
* add a note of where in the monorepo the tool is: or at least mention its in the `nym-wallet` directory

Reply:
* final thing - typo in `encryted`